test/integration/helpers/Env.cs

```diff
@@ -15,15 +15,15 @@ public class Env
 
         private static void Init()
         {
+            if (_initialized) return;
+
             _env = new Dictionary<string, JsonElement>
             {
                 { "DEV", JsonDocument.Parse("true").RootElement }, 
                 { "isRemoteAppiumServer", JsonDocument.Parse("false").RootElement }, 
                 { "remoteAppiumServerUri", JsonDocument.Parse("\"http://localhost:4723\"").RootElement }
             };
 
-            if (_initialized) return;
-
             try
             {
                 _initialized = true;
```

Comment:
should the flag be set at the very end of this method?

Reply:
Makes sense. fixed